src/components/FeaturedProject.tsx

```diff
@@ -1,45 +1,39 @@
-import React, { useState } from "react";
+import React from "react";
+import HoverButton from "components/HoverButton/HoverButton";
 import * as S from "./FeaturedProject.styles";
+import { ButtonMode } from "./HoverButton/HoverButton.styles";
 
 interface featuredProjectProps {
-  image?: string;
-  github: string;
-  description: string;
-  name: string;
+    image?: string;
```

Comment:
Should not be optional. Also I suggest calling it "demoUrl" or "recording" or something along those lines